dappnode_package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "vipnode.dnp.dappnode.eth",
-  "version": "0.0.1",
-  "description": "https://vipnode.org - Economic incentive for running Ethereum full nodes. The goal is to allow the Ethereum network to remain decentralized by creating a financial marketplace for more people to run full nodes and serve native light clients.",
-  "avatar": "/ipfs/Qmen3srZXEHncMM2RPsgVKkPbkJPTMN9SNFVEAQQY4a7Nf",
+  "version": "0.0.3",
+  "upstreamVersion": "2.0.1",
```

Comment:
Might be worth updating to the latest upstream version. I also opened https://github.com/dappnode/DAppNodePackage-vipnode/pull/7